_pages/en_US/installing-boot9strap-(menuhax).txt

```diff
@@ -0,0 +1,89 @@
+---
+title: "Installing boot9strap (menuhax)"
+---
+
+{% include toc title="Table of Contents" %}
+
+{% capture technical_info %}
+<summary><em>Technical Details (optional)</em></summary>
+
+We use BannerBomb3 to install menuhax67 which lets us access the Homebrew Launcher. From there we launch uloader which launches the SafeB9SInstaller. 
+
+Info about uloader [here](https://github.com/zoogie/uloader), info about menuhax67 [here](https://github.com/zoogie/menuhax67).
```

Comment:
Should expand this into full sentences:

See [here] for information about uloader and [here] for information about menuhax67. (or something similar)